public/index.pug

```diff
@@ -29,22 +29,40 @@ html(lang="en-US")
+        li(class="header__list-item")
+          a(class="header__list-icon" title="Legal disclaimer" href="https://github.com/simple-icons/simple-icons/blob/master/README.md" rel="noopener") 
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
+              path(d="m23.9 9.7-3.54-7.89-3.2e-4 -6.41e-4 -0.0046-0.0102c-0.0119-0.0264-0.026-0.0512-0.0415-0.0748-0.0042-0.0064-0.0092-0.0122-0.0136-0.0183-0.014-0.0194-0.029-0.0377-0.0453-0.0549-0.0064-0.0067-0.0129-0.0134-0.0196-0.0198-0.0179-0.017-0.0368-0.0327-0.0569-0.047-0.0052-0.0038-0.01-0.0081-0.0155-0.0118-0.0252-0.0167-0.0519-0.031-0.0797-0.0432-0.0079-0.0035-0.0161-0.0061-0.0242-0.0092-0.0218-0.0083-0.0439-0.0153-0.0668-0.0207-0.0094-0.0022-0.0187-0.0045-0.0282-0.0062-0.0313-0.0058-0.0631-0.0096-0.0957-0.0096h-6.85c-1.02-1.52-1.02-1.54-2 1.06e-4h-6.86c-0.0326 0-0.0643 4e-3 -0.0957 0.0096-0.00961 0.0017-0.0188 4e-3 -0.0282 0.0062-0.0229 0.0054-0.0451 0.0124-0.0667 0.0207-0.00812 0.0031-0.0163 0.0057-0.0242 0.0092-0.0278 0.0123-0.0545 0.0265-0.0797 0.0432-0.00545 0.0036-0.0103 0.0079-0.0155 0.0118-0.0201 0.0143-0.039 0.0299-0.0569 0.047-0.00684 0.0064-0.0132 0.013-0.0198 0.0198-0.0163 0.0172-0.0313 0.0356-0.0453 0.0549-0.00449 0.0062-0.0094 0.012-0.0136 0.0183-0.0156 0.0236-0.0297 0.0484-0.0415 0.0748l-0.00459 0.0102-3.2e-4 6.4e-4 -3.55 7.91c-0.06 0.0864-0.0958 0.191-0.0958 0.304 0 2.28 1.86 4.14 4.14 4.14s4.14-1.86 4.14-4.14c0-0.113-0.0357-0.218-0.0958-0.304l-3.25-6.37 6.07-0.0229v18.2c-2.55 0.294-7.01 0.381-7 2.5h16c-0.0011-2.03-4.48-2.27-7-2.5v-18.1l5.69-0.0206-2.92 6.49c-8.55e-4 0.0019-0.0013 0.0038-0.0021 0.0058-0.0025 0.0058-0.0043 0.0118-0.0066 0.0176-0.0096 0.0245-0.0173 0.0494-0.0232 0.0746-0.0015 0.0066-0.0034 0.013-0.0047 0.0196-0.0059 0.0306-0.0089 0.0615-0.0093 0.0925 0 0.0029-8.54e-4 0.0054-8.54e-4 0.0083 0 2.28 1.86 4.14 4.14 4.14 2.28 0 4.14-1.86 4.14-4.14 0-0.126-0.0457-0.241-0.119-0.332z")
+        li(class="header__list-item")
+          a(class="header__list-icon" title="About Simple Icons" href="https://github.com/simple-icons/simple-icons/blob/master/README.md" rel="noopener") 
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
+              path(d="M12 0C8.688 0 6 2.688 6 6s2.688 6 6 6c1.066 0 2.1171.2862 3.0371.8262 2.858 1.678 3.8167 5.3539 2.1387 8.2129h1.6797a7.4925 7.4925 0 0 0 .6425-3.0293c.003-4.144-3.356-7.5048-7.498-7.5098-2.484 0-4.5-2.016-4.5-4.5S9.516 1.5 12 1.5s4.5 2.016 4.5 4.5H18c0-3.312-2.688-6-6-6zm0 3c-1.597.04-2.8799 1.3259-2.9219 2.9219C9.0351 7.5799 10.343 8.957 12 9c1.597-.04 2.8799-1.3259 2.9219-2.9219C14.9649 4.4201 13.656 3.043 12 3zm0 1.5c.828 0 1.5.672 1.5 1.5v.002c0 .828-.672 1.5-1.5 1.5-.83 0-1.5-.673-1.5-1.502 0-.83.67-1.5 1.5-1.5zM7.5 15v1.5H9v6H4.5V24h15v-1.5H15v-6h1.5V15h-9zm3 1.5h3v6h-3v-6zm-6 1.4707V18a7.418 7.418 0 0 0 .6445 3.039h1.6836C6.3001 20.147 6 19.11 6 18v-.0293H4.5Z")
```

Comment:
What's the motivation for adding this link? I'm not sure what purpose it serves in addition to the GitHub link

Reply:
It's different to point to the repository than to point to the README file.
Besides, the idea is to point to the contributing page, the contributors list, maybe put the legal disclaimer in there as well.

Reply:
The menu now has the background color effect as discussed.

Reply:
Okay, but that's unrelated to this comment thread.

Reply:
Link to README has been removed